e2e/terraform/main.tf

```diff
@@ -36,47 +38,6 @@ resource "google_compute_instance" "vm_instance" {
     access_config {
     }
   }
-  provisioner "file" {
-    source      = "../provision"
-    destination = "/home/ubuntu/provision"
-    connection {
-    host        = self.network_interface[0].access_config[0].nat_ip
-    type        = "ssh"
-    private_key = "${file(var.ssh_prv_key)}"
-    user        = "${var.ansible_user}"
-    agent       = false
-  }
-
-  }
-
-  provisioner "file" {
-    source      = "/etc/nephio/nephio.yaml"
-    destination = "/home/ubuntu/nephio.yaml"
-    connection {
-    host        = self.network_interface[0].access_config[0].nat_ip
-    type        = "ssh"
-    private_key = "${file(var.ssh_prv_key)}"
-    user        = "${var.ansible_user}"
-    agent       = false
-  }
-
-  }
-
-  provisioner "remote-exec" {
-    connection {
-    host        = self.network_interface[0].access_config[0].nat_ip
-    type        = "ssh"
-    private_key = "${file(var.ssh_prv_key)}"
-    user        = "${var.ansible_user}"
-    agent       = false
-  }
-    inline = [
-               "cd provision/",
-               "chmod +x gce_install_sandbox.sh",
-               "export DEBUG=true",
-               "timeout --preserve-status 30m ./gce_install_sandbox.sh"
```

Comment:
@johnbelamaric why those instructions were removed?

Reply:
We shouldn't need this anymore. We instead launch the VM with an startup-script-url and nephio-run-e2e metadata, and it takes care of it from there.

Reply:
But I haven't been able to test the terraform changes, so we could pull them from this PR and do that in another PR. But the idea is that you can launch the VM and run the whole e2e suite with a single gcloud command, as described in the README. We can leverage that for the terraform too.

Reply:
I think that we can use that as another alternative to provision a development/test environment (besides Vagrant and others) but not as replacement for e2e process, mainly because it's not using the PR's code for its execution.

Reply:
ah, yes, good point. I will remove this and we will address it at a later time.

Reply:
I took your suggestions, but would you prefer I remove the changes to the terraform from the PR?

Reply:
I'm not a Terraform expert, but I think that we can use a conditional here to reuse as much as possible. In that way, one path can be used by e2e and the other for testing/developing 

/cc @radoslawc

Reply:
> But the idea is that you can launch the VM and run the whole e2e suite with a single gcloud command, as described in the README. We can leverage that for the terraform too.

Terraform advantage here was that it stores all entities that were created in its state files (that can be for example pushed to a bucket). So when they are no longer needed those resources can be easily removed. For now we're spinning up single VM, so that's not crucial, but if there's going to be a need for more complicated setups this might come handy.